src/components/ViewPCOL/Details.tsx

```diff
@@ -9,6 +9,12 @@ const ViewPCOLDetails = () => {
   const pcol = usePCOL(program, pcolId);
   const contracts = useContracts(program);
 
+  const contractor =
```

Comment:
Suggest doing something similar for the Contract -- and handle case where the contract isn't establisted yet, so that it doesn't do a dash at the end ("No Established Contract -")